integ-test/src/test/java/org/opensearch/sql/sql/DateTimeFunctionIT.java

```diff
@@ -151,6 +216,93 @@ public void testDateSub() throws IOException {
     verifyDataRows(result, rows("2020-09-15"));
   }
 
+  @Test
+  public void testDateTime() throws IOException {
```

Comment:
What is the importance of each test case? You could break them out into separate tests (🔥) or at least document each one.